program/milestones/CV32E40P/RTL_Freeze_v1.0.0/DesignChecklist.md

```diff
@@ -3,31 +3,31 @@ An item is "signed off" once the `Signed-off By` and `Sign-off Date` cells are f
+| Coding rules     | Module and File names                             | No two modules in the design have the same name. Each module name shall be prefixed with the project name (e.g. cv32e40p_<module>) Each module name shall match the file name using the system verilog suffix ".sv" (e.g. cv32e40p_controller module located in cv32e40p_controller.sv file) Each system verilog file shall contain only one module definition                                                                      | davide@openhwgroup.org | 2020-12-02 | Except for the cv32e40p_register_file who is defined both in the cv3240p_register_file_ff.sv and cv32e40p_register_file_latch.sv file. Only one can be compiled. By default the ff version is selected as reported in the manifest file.    |
+| Coding rules     | Implementation HDL                                | The design does not mix HDLs.                                                                                                                                                                                                                                                                                                                                                                                                       | davide@openhwgroup.org | 2020-12-02 |                                                                                                                                                                                                                                             |
+| Coding rules     | Low technology dependence                         | In IP designs, technology-specific blocks (mainly RAM cells) are accessible and easily replaced by blocks targetting other ASIC or FPGA technologies.                                                                                                                                                                                                                                                                               | davide@openhwgroup.org | 2020-12-02 | The cv32e40p_sim_clock_gate.sv contains a tech-specific block that can be easily replaced.                                                                                                                                                  |
```

Comment:
Actually it does _not_ contain a tech-specific block, but it _needs_ a tech-specific block that needs to be provided by the user.